tests/bgp/test_bgp_authentication.py

```diff
@@ -145,7 +145,7 @@ def test_bgp_peer_group_password(setup):
 
     # mismatch peer group passwords
     cmd = 'vtysh -n {} -c "config" -c "router bgp {}" -c "neighbor {} password {}" -c "neighbor {} password {}" \
-          -c "end"'.format(setup['neigh_asic_index'], setup['dut_asn'], setup['peer_group_v4'], mismatch_pass,
+          -c "end"'.format(setup['asic_index'], setup['dut_asn'], setup['peer_group_v4'], mismatch_pass,
```

Comment:
can you add check sth like:
ns = "-n " + setup['asic_index'] if asic_index, else ""
cmd = 'vtysh '+ ns + ' -c ....'

to avoid the case where single asic has None as asic_index OR neigh_asic_index

Reply:
here probably keep original change, add a check, and change line 151 to run on neighborhost